modules/dreamview/conf/hmi_modes/mkz_64beam.pb.txt

```diff
@@ -0,0 +1,160 @@
+cyber_modules {
+  key: "Camera"
+  value: {
+    dag_files: "/apollo/modules/drivers/camera/dag/camera.dag"
+    dag_files: "/apollo/modules/drivers/video/dag/video.dag"
+  }
+}
+cyber_modules {
+  key: "Canbus"
+  value: {
+    dag_files: "/apollo/modules/canbus/dag/canbus.dag"
+  }
+}
+cyber_modules {
+  key: "Control"
+  value: {
+    dag_files: "/apollo/modules/control/dag/control.dag"
+  }
+}
+cyber_modules {
+  key: "GPS"
+  value: {
+    dag_files: "/apollo/modules/drivers/gnss/dag/gnss.dag"
+  }
+}
+cyber_modules {
+  key: "Guardian"
+  value: {
+    dag_files: "/apollo/modules/guardian/dag/guardian.dag"
+  }
+}
+cyber_modules {
+  key: "Localization"
+  value: {
+    dag_files: "/apollo/modules/localization/dag/dag_streaming_msf_localization.dag"
```

Comment:
default to rtk localization as of now since MSF has frequency issue now in fix

Reply:
OK